app/model/abc.py

```diff
@@ -108,3 +108,10 @@ def process_bind_param(self, value, dialect):
     def process_result_value(self, value, dialect):
         return self.choices[value]
 
+
+class Base(db.Model, BaseModel):
+    __abstract__ = True
+
+    created_at = db.Column(db.DateTime, default=db.func.now())
```

Comment:
Shouldn't we enforce `nullable=False` to the creation timestamps.

Reply:
FIx.